crates/astria-composer/src/config.rs

```diff
@@ -30,6 +30,12 @@ pub struct Config {
     /// Private key for the sequencer account used for signing transactions
     #[serde(serialize_with = "serialize_private_key")]
     pub private_key: SecretString,
+
+    /// Block time in milliseconds
+    pub block_time: u64,
+
+    /// Max bytes to send in a single sequencer `SignedTransaction` (sum of all `SequenceAction`s)
```

Comment:
```suggestion
    /// Max bytes to send in a single sequencer `SignedTransaction`
```
since it's actually the encoded size

Reply:
ok, i see now that in the `SizedBundle` it submits once the sum of the sequence::Action lengths is > this size. however this doesn't take into account signature + pubkey size, nonce, encoding size, etc. that's put into the `SignedTransaction`, so this needs to be less than 250kb by default

Reply:
fixed comments